.travis/install.test.sh

```diff
@@ -33,11 +33,11 @@ R -e 'install.packages(c("forge"), quiet = TRUE)'
 CPYTHON_VERSION=$(python3 -c 'import sys; print(str(sys.version_info[0])+str(sys.version_info[1]))')
 python3 -m pip install wheelhouse/*-cp${CPYTHON_VERSION}-*.whl
 python3 -m pip install -q pytest boto3 pyarrow==0.11.1 pandas==0.19.2
-(cd tests && python3 -m pytest .)
+(cd tests && python3 -m pytest --import-mode=append .)
```

Comment:
Previous CI install.test job failed with:
```
_____________________ ERROR collecting tests/test_arrow.py _____________________
test_arrow.py:35: in <module>
    from tensorflow_io.arrow.python.ops import arrow_dataset_ops
/v/tensorflow_io/arrow/__init__.py:24: in <module>
    from tensorflow_io.arrow.python.ops.arrow_dataset_ops import ArrowDataset
/v/tensorflow_io/arrow/python/ops/arrow_dataset_ops.py:30: in <module>
    arrow_ops = _load_library('_arrow_ops.so')
/v/tensorflow_io/__init__.py:54: in _load_library
    raise NotImplementedError("unable to open file: ", filename)
E   NotImplementedError: ('unable to open file: ', '_arrow_ops.so')
```
So it seems to be picking up tensorflow_io from the source directory and not the installation. I could reproduce this locally and adding the option `--import-mode=append`. I'm not sure if this had anything to do with my changes or not.

Reply:
Ahh, figured it out. It was the addition of `tests/__init__.py`.